aiokafka/consumer/group_coordinator.py

```diff
@@ -402,8 +410,18 @@ async def _perform_assignment(
             'Invalid assignment protocol: %s' % assignment_strategy
         member_metadata = {}
         all_subscribed_topics = set()
-        for member_id, metadata_bytes in members:
+        group_instance_id_mapping = {}
```

Comment:
Not sure what this is used for as it gets replaced by `group_instance_id` a few lines down and that it only serves for AbstractStaticPartitionAssignor. Couldn't we directly use `group_instance_id` instead ?